### ISSUE TITLE
"username@tox-central-server" is really needed?

### ISSUE THREAD

**Issue by verycrypt** [irungentoo/toxcore#1568]
> Removing the need to rely on central authorities to provide messenger services
> https://tox.chat/faq.html

If so, why https://github.com/Antox/Antox and https://github.com/tux3/qTox use
toxme.io as a default?

toxme.io
toxme.xx (operated by other people, I forgot actual address)
etc.
---

Why don't you just disable toxme.io, and use tox.chat? (because using many domains make
user hard to find "official" data.)

e.g.,
https://userbook.tox.chat/ - where user can upload their long ToxID(with their permission)

(!) Antox force users to register themselves - because if they push "Register" button,
they will register automatically without notice!

**Comment by Impyy** [comment]
We don't want to endorse centralized solutions for name lookups. That's why it is on a different domain, hosted by a third party and doesn't make an appearance on our official website.

I agree that Antox doesn't make it clear enough that it registers you at toxme.io. but that is not a toxcore issue, please file it at the Antox repo.

**Comment by verycrypt** [comment]
> We don't want to endorse centralized solutions for name lookups. That's why it is on a different domain

Uh... if so, how about *just* using https://userbook.tox.chat/ and call users
"username@pine.tox" (no domain name) ?

---
username ... user's choice
@ ... at,
pine (selectable from drop-down lists at registration webpage)
.
tox (end with tox)
---

I wrote this issue because
1. Current "user@tox.domain" is a central cervice, controlled by tox.domain.
2. Tox user need to know about "toxme.io".
Expected questions:
"What is toxme.io? I downloaded Tox app from 'tox.chat'!"
"My friend asked me to add hisid@toxchat.xx. Can I connect it from my@toxme.io?"
